### PR TITLE
added ingresses-only flag

### LINKED ISSUE #67: Add a `ingresses-only` flag
**What would you like to be added**:
Add a `ingresses-only` flag to scope the tool to only convert resources without looking for providers CRDs

**Why this is needed**:
Provider specific support is in early stage support now with only few providers support the conversion. Users might want to scope the tool invocation to only read and convert their ingresses

/label good-first-issue
/help-wanted

### PR DESCRIPTION
**Pull request by camperjett**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
this adds a new flag `ingresses-only` to scope the tool to only convert resources without looking for providers CRDs

**Which issue(s) this PR fixes**:
Fixes #67 

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```
/cc @LiorLieberman

### CODE REVIEW

**Review by levikobi — CHANGES_REQUESTED** [review]
Thank you for your contribution @camperjett!

Removing the gateway objects is not exactly what we had in mind here. What we intended to do was to not translate provider-specific functionality by not running provider converters.
Maybe you can take a look at https://github.com/kubernetes-sigs/ingress2gateway#usage for some reference.

Let me know if you have any questions.

**Comment by camperjett** [comment]
Okay. This is similar to passing empty list of providers in function `i2gw.ToGatewayAPIResources(...)` right? Should I do that?

**Comment by levikobi** [comment]
> Okay. This is similar to passing empty list of providers in function `i2gw.ToGatewayAPIResources(...)` right? Should I do that? (In file cmd/print.go, Inside function `PrintGatewaysAndHTTPRoutes(...)`)

Yes that seems about right

One more thing that comes to mind, this flag should probably be mutually exclusive with the providers flag

File: cmd/print.go [hunk 1]
```diff
@@ -182,6 +190,7 @@ if specified with --namespace.`)
 	cmd.Flags().StringSliceVar(&pr.providers, "providers", i2gw.GetSupportedProviders(),
 		fmt.Sprintf("If present, the tool will try to convert only resources related to the specified providers, supported values are %v", i2gw.GetSupportedProviders()))
 
+	cmd.Flags().BoolVar(&pr.ingressesOnly, "ingresses-only", false, "If present, the tool will only print the converted Ingresses.")
```
Comment (levikobi) [inline cmd/print.go:193]: Should be mutually exclusive with the providers flag

**Comment by camperjett** [comment]
PTAL at 
#71

**Pull request closed without merging**